test-app-sitebricks.yaml

```diff
@@ -51,9 +51,9 @@ paths:
           description: "invalid id of user"
```

Comment:
yes, it is a demo app, one of the points of which I think is to try out the app engine and datastore, which is why I think it is a good idea to have the datastore generate your id's at least for one of your entities